test/core/channel/BUILD

```diff
@@ -87,7 +87,6 @@ grpc_cc_test(
     external_deps = [
         "gtest",
     ],
-    flaky = True,  # TODO(b/151792070)
```

Comment:
I'd actually wait a few days with removing the flaky flag (once we are sure that the test has actually stopped failing in all configurations). It seems like a good practice.

Reply:
Agreed. WIll do!